Sources/Plasma/FeatureLib/pfGLPipeline/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+set(pfGLPipeline_SOURCES
+    plGLDevice.cpp
+    plGLDeviceRefs.cpp
+    plGLPipeline.cpp
+)
+
+set(pfGLPipeline_HEADERS
+    plGLDevice.h
+    plGLDeviceRef.h
+    plGLPipeline.h
+    pfGLPipelineCreatable.h
+)
+
+plasma_library(pfGLPipeline SOURCES ${pfGLPipeline_SOURCES} ${pfGLPipeline_HEADERS})
+target_link_libraries(pfGLPipeline
+    PUBLIC
+        CoreLib
+        pnKeyedObject
+        pnModifier
+        pnNucleusInc
+        plGImage
+        plPipeline
+        plProgressMgr
+        plScene
+        plStatusLog
+    PRIVATE
+        pnMessage
+        pnNetCommon
+        pnSceneObject
+        plAudio
+        plAvatar
+        plClientResMgr
+        plDrawable
+        plGLight
+        plIntersect
+        plMessage
+        plNetClient
+        plResMgr
+        plSurface
+        pfCamera # plCaptureRender, plDynamicEnvMap, plDXPipeline
+        pfGameGUIMgr # plCaptureRender
+        OpenGL::GL
+    INTERFACE
+        pnFactory
+)
```

Comment:
As above, have all of these been validated?

Reply:
Not validated